### ISSUE TITLE
Forwarded messages are not announced as so when arrowing through lists

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2200]
Steps:
1. Focus a forwarded message with the arrows.

Expected: The message should read something like "Forwarded message from original_sender. Sender. Message".
Actual: Only "Sender. Message" is announced. The first bit is announced if I press tab, or with object navigation as always.

**Comment by FrayxRulez** [comment]
This was implemented
